shared/wallets/transaction/index.js

```diff
@@ -37,7 +38,20 @@ const CounterpartyIcon = (props: CounterpartyIconProps) => {
     case 'stellarPublicKey':
       return <Icon type="icon-placeholder-secret-user-48" style={{height: size, width: size}} />
     case 'otherAccount':
-      return <Icon type="icon-wallet-to-wallet-48" style={{height: size, width: size}} />
+      return (
+        <Box2
+          alignSelf="flex-start"
+          direction="horizontal"
+          style={collapseStyles([styles.transferIconContainer, {width: size}])}
+        >
+          <Icon
+            color={globalColors.purple2}
+            sizeType={props.detailView ? 'Bigger' : 'Big'}
+            style={collapseStyles([!props.detailView && styles.transferIcon])}
+            type="iconfont-wallet-transfer"
+          />
+        </Box2>
```

Comment:
Can we achieve the same result by using `padding="xtiny"` instead of this added Box2?

Reply:
Maybe, but we also need the icon to be a set size and a container with a different width to line it up with the other icons.